src/tasks/AndroidAppBuilder/Templates/monodroid-coreclr.c

```diff
@@ -34,11 +36,15 @@ Java_net_dot_MonoRunner_freeNativeResources (JNIEnv* env, jobject thiz);
 
 /********* implementation *********/
 
-static const char* g_bundle_path = NULL;
+static char* g_bundle_path = NULL;
 static const char* g_executable_path = NULL;
 static unsigned int g_coreclr_domainId = 0;
 static void* g_coreclr_handle = NULL;
 
+#define MAX_MAPPED_COUNT 50 // Arbitrarily 'large enough' number
```

Comment:
nit: I suggest bumping this to at least 256? 

It is true that we do not run libraries tests yet with CoreCLR but trimmed `System.Runtime.Tests` has 83 assemblies. On the other hand, if we disable trimming even for sample app the count is 172.

Reply:
```suggestion
#define MAX_MAPPED_COUNT 256 // Arbitrarily 'large enough' number
```

Reply:
"Good enough" for me.